scripts/setup-kind-multicluster.sh

```diff
@@ -47,7 +47,17 @@ containerdConfigPatches:
     endpoint = ["http://${registry_name}:${registry_port}"]
 nodes:
 - role: control-plane
-$(for ((i=0; i<$num_workers; i++)); do echo "- role: worker"; done)
+$(for ((i=1; i<=$num_workers; i++)); do
+cat << EOF2
+- role: worker
+  kubeadmConfigPatches:
+    - |
+      kind: JoinConfiguration
+      nodeRegistration:
+        kubeletExtraArgs:
+          node-labels: "rack=rack$i"
```

Comment:
Please note: this change will require you to upgrade kind to its latest version, 0.11.1.